<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelDatasMonster</class>
 <widget class="QWidget" name="PanelDatasMonster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>1238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="PanelDatasCharacter" name="panelDatasCharacter" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>530</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>530</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxRewards">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>500</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>500</height>
        </size>
       </property>
       <property name="title">
        <string>Rewards</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
            <item>
             <widget class="QLabel" name="labelExperience">
              <property name="text">
               <string>Experience:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="PanelProgressionTable" name="panelProgressionTableRewardExp" native="true"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="labelCurrencies">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Currencies:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout"/>
            </item>
            <item>
             <widget class="QTabWidget" name="tabWidgetCurrencies">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="currentIndex">
               <number>-1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="labelLoots">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Loots:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="WidgetSuperTree" name="treeViewLoots">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <attribute name="headerMinimumSectionSize">
               <number>35</number>
              </attribute>
              <attribute name="headerDefaultSectionSize">
               <number>35</number>
              </attribute>
              <attribute name="headerStretchLastSection">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxActions">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Actions</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="WidgetSuperTree" name="treeViewActions">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>150</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetSuperTree</class>
   <extends>QTreeView</extends>
   <header>widgetsupertree.h</header>
  </customwidget>
  <customwidget>
   <class>PanelDatasCharacter</class>
   <extends>QWidget</extends>
   <header>paneldatascharacter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PanelProgressionTable</class>
   <extends>QWidget</extends>
   <header>panelprogressiontable.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
